themes-default/slim/views/addShows_addExistingShow.mako

```diff
@@ -171,7 +172,7 @@ window.app = new Vue({
                 const seriesToAdd = [dir.selectedIndexer, dir.path, seriesId, dir.metadata.seriesName]
                     .filter(i => typeof(i) === 'number' || Boolean(i)).join('|');
 
-                formData.append('shows_to_add', encodeURIComponent(seriesToAdd));
+                formData.append('shows_to_add', seriesToAdd);
```

Comment:
@OmgImAlexis is this okay?

Reply:
tl;dr: I think it should be okay, as long as you're consistent (removing the "decode" on the receiving end)

IIRC I just left it as it was on the old code, which _instead_ of sending a `POST` request with all the data, _it created_ a URL to navigate to (`GET` with query params).

**Original:** https://github.com/pymedusa/Medusa/commit/c408a6a0e0b65add222a8ebd2e7512f8a1958f21#diff-03bf01eea7b4fadd7983b2e8bb1f43daR108
**Switch to `POST`**: https://github.com/pymedusa/Medusa/commit/0e0449196eeef5138405dad9199f63cb279f2d97